docs/examples/username-pass.yaml

```diff
@@ -9,6 +9,8 @@ metadata:
   annotations:
 # description - can not be a label as spaces are not allowed
     "jenkins.io/credentials-description" : "credentials from Kubernetes"
+# job/folder scope - optional - individual items must begin and end with '/'
+    jenkins.io/credentials-item-group: ["/job/thisIsJobA/", "/job/thisIsJobB/job/nestedJob/"]
```

Comment:
why use the URL path as opposed to the actual path for the Item?

https://javadoc.jenkins.io/hudson/model/Item.html#getFullName()